### DISCUSSION TITLE
Account reports including initial balance in odoo 17 accounting community edition??

### DISCUSSION THREAD

**Discussion by shyambista1** [community/community#135886] [Other Feature Feedback, Questions, & Ideas]
I'm looking for solutions or free modules for Odoo 17 Accounting Community Edition that include initial balances in their account reports. Specifically, I'm interested in finding modules that ensure initial balances are accurately reflected in reports like balance sheets or trial balances. If anyone has recommendations or has come across any community modules that handle this effectively, your input would be greatly appreciated. Thanks!

**Comment by LiteBrite82** [comment]
Hi @shyambista1 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by BhathiyaPrasad** [comment]
Hello, did you find an answer for this, actually I found one
